system/ee/ExpressionEngine/Model/Category/Category.php

```diff
@@ -68,7 +68,7 @@ class Category extends ContentModel
 
     protected static $_validation_rules = array(
         'cat_name' => 'required|noHtml|xss',
-        'cat_url_title' => 'required|alphaDash|unique[group_id]',
+        'cat_url_title' => 'required|alphaDash|unique[site_id]',
```

Comment:
Unfortunately I think this is a breaking change.  It's been long standing that EE can have duplicate categories in different groups, but not the same.